### ISSUE TITLE
ability to go install "github.com/anchore/grype"

### ISSUE THREAD

**Issue by dreamcat4** [anchore/grype#568]
**What happened**:

```sh
Installing tools from tools.go
go install "github.com/anchore/grype"
# github.com/anchore/syft/internal/formats/spdx22tagvalue
../.cache/go/pkg/github.com/anchore/syft@v0.34.0/internal/formats/spdx22tagvalue/to_format_model.go:186:4: unknown field 'PackageChecksumSHA1' in struct literal of type spdx.Package2_2
../.cache/go/pkg/github.com/anchore/syft@v0.34.0/internal/formats/spdx22tagvalue/to_format_model.go:187:4: unknown field 'PackageChecksumSHA256' in struct literal of type spdx.Package2_2
../.cache/go/pkg/github.com/anchore/syft@v0.34.0/internal/formats/spdx22tagvalue/to_format_model.go:188:4: unknown field 'PackageChecksumMD5' in struct literal of type spdx.Package2_2
```

**What you expected to happen**:

Ability to install via `go install`. I know that might sound a bit naiive. However it would be really helpful to support that mechanism. Which is supposed to be the official mechanism of go. It makes managing installations and updating of your tool so much easier for me.  If it uses the same mechanism as all my other `go` based tools.

Of course there might be other challenges related to what is required to install `grype` and `syft` properly. In terms of other non-go platform dependencies which sit outside of go. Just do what you can if it's not too much to ask i guess? And also if happens to align with your own specific go based ethos / philosophies

**How to reproduce it (as minimally and precisely as possible)**:

`go install "github.com/anchore/syft"`

**Environment**:

- OS (e.g: `cat /etc/os-release` or similar):

```sh
$ cat /etc/os-release
NAME="Ubuntu"
VERSION="21.04 (Hirsute Hippo)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 21.04"
VERSION_ID="21.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=hirsute
UBUNTU_CODENAME=hirsute
```

**Comment by luhring** [comment]
Hi @dreamcat4, I just tried to reproduce this using Docker, for the sake of a clean environment. I'm not able to reproduce this yet.

On my end, it looks like the `go install` approach is working.

Here's what I see:
```
$ docker run --rm golang:1.17 go install github.com/anchore/grype@latest
go: downloading github.com/anchore/grype v0.28.0
go: downloading github.com/anchore/grype-db v0.0.0-20211207213615-1bcbb779ee96
go: downloading github.com/anchore/stereoscope v0.0.0-20211222141827-6e663afeef5
[...]
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
go: downloading github.com/go-restruct/restruct v1.2.0-alpha
$
```

Can you share the full command you ran, and the full output you received?

**Comment by dreamcat4** [comment]
wow that is good news to hear. thanks for checking!

in terms of my own local environment. where this error is taking place. hmm well lets see now


```sh
$ go version
go version go1.17.5 linux/amd64
```

ah ok. I can see here that you appent `@latest` version specifier. For latest released tagged version. I neglected to do that over here....

So that worked all fine now. Therefore maybe the error message received was to do with something more recent additions into the `master` or `main` development branch? Or otherwise I have seen situations (in other go project). When the configuration setup of the go modules / dependancies were specified in a fashion that was no longer compatible with the new golang specs (for newer versions of go, they keep changing certain versioning scheme such as `/v1` or `/v3` etc)

Either way my own individual problem here is now solved by appending `@latest` into my `tools.go` and they are now installed and seems working fine. Thank you for the concerns, and the 2nd oversight to find and correct the issue. Appreciates,
